herddb-core/src/main/java/herddb/core/system/SyscolumnsTableManager.java

```diff
@@ -71,7 +72,8 @@ public SyscolumnsTableManager(TableSpaceManager parent) {
                         "column_name", c.name,
                         "ordinal_position", pos++,
                         "is_nullable", nonNullCType ? DatabaseMetaData.columnNoNulls : DatabaseMetaData.columnNullable,
-                        "data_type", data_type,
+                        "data_type",  ColumnTypes.sqlDataType(c.type),
+                        "type_name", data_type,
```

Comment:
Can you change data_type var to another name? Having a data_type record field that not report data_type var and a type_name that report data_type value is a little misleading

Reply:
@diegosalvi  "data_type" is already here, we can't change it without breaking compatibility with old clients

Reply:
@amitvc in this file we already had 'data_type' 
>  String data_type = ColumnTypes.typeToString(c.type);
Is that variable no more used ?

As said, we shouldn't change the behaviour for "data_type", can you explain better your change here ?

Reply:
I didn't meant to change "data_type" string but just rename data_type variable to type_name variable (such variable has been created 5-6 lines before and a rename won't break anything)... is just strange assign data_type value to "type_name" string and ColumnTypes.sqlDataType(c.type) to "data_type" string. [Otherwise the next time I'll stumble there there I'll need time to check if is an error or is the expected behaviour :(]

Reply:
Good catch. It's my bad. I should have renamed the variable.